server/templates/base.html

```diff
@@ -90,14 +90,6 @@
             </div>
             {% endif %}
             <a href="{{ url_for('static.homepage') }}">{{ NAME }}</a>
-            {% if subpage_title %}
```

Comment:
@chejennifer - What is the subpage that you are referring too, maybe yes, but want to see before I vote ;-)

Reply:
this piece of code refers to this part of the header: https://screenshot.googleplex.com/9556CVikX5ZHQat

& usually clicking it will take the user back to the landing page for that tool

Reply:
Thanks, also removed subpage_title from other pages on main DC and biomedical (but left in custom dc, which uses custom dc base template)

Change is already in description, I had called it subtitle instead of subpage_title before.